src/content_sao.cpp

```diff
@@ -1193,6 +1193,10 @@ void PlayerSAO::setPos(const v3f &pos)
 	if(isAttached())
 		return;
 
+	// Send mapblock of target location
+	v3s16 blockpos = v3s16(pos.X / MAP_BLOCKSIZE, pos.Y / MAP_BLOCKSIZE, pos.Z / MAP_BLOCKSIZE);
+	m_env->getGameDef()->SendBlock(m_peer_id, blockpos);
```

Comment:
How often is setPos called? Ie: is it called when the client sends a position update

Reply:
No, only when the server, though lua, sends a location change (teleport)

Reply:
This includes /home and resurrection.